package.json

```diff
@@ -16,7 +16,7 @@
     "storybook": "start-storybook --static-dir ./public -p 6006",
     "test": "npm-run-all format:check lint test:jest test:playwright",
     "test:jest": "jest tests src --no-cache",
-    "test:playwright": "npx playwright test",
+    "test:playwright": "npx playwright install && npx playwright test",
```

Comment:
This doesn't seem quite correct to me, you shouldn't need to install playwritght to run the tests. It sounds like the probably is that there's a mismatch between the playwright version we have installed and the docker image defined here. 

It looks like playwright recommends that you keep the version of the docker container in sync with the version of the library you have installed:

https://github.com/microsoft/playwright/issues/10410#issuecomment-972996397

However, I honestly don't know how to keep the [docker container definition](https://github.com/mozilla/glean-dictionary/blob/9f7e9a34e658cb40d087f8cc97936f84860870d3/.circleci/config.yml#L11) in sync with what's in our package.json offhand, since dependabot doesn't send updates for the former.

I'd suggest we do two things here:

1. Add the `npx playwright install` step to the circleci configuration above.
2. Add a small note to the README.md about `npx playwright install`  and `npx playwright test` (maybe just below [our notes about storybook](https://github.com/mozilla/glean-dictionary#storybook))